zeppelin-interpreter/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -780,6 +780,7 @@ public String getZeppelinNotebookCronFolders() {
         "zeppelin.interpreter.lifecyclemanager.timeout.checkinterval", 6000L),
     ZEPPELIN_INTERPRETER_LIFECYCLE_MANAGER_TIMEOUT_THRESHOLD(
         "zeppelin.interpreter.lifecyclemanager.timeout.threshold", 3600000L),
+    ZEPPELIN_INTERPRETER_INTERPOLATION("zeppelin.interpreter.interpolation", false),
```

Comment:
It it not necessary to put it in ZeppelinConfiguration & zeppelin-env.sh. Properties in them are global properties, since it is interpreter level property, we only need to put it in interpreter-setting.json

Reply:
Thanks @zjffdu that was very helpful. 

The following changes have been made:
1) Removed global property definitions from _ZeppelinConfiguration_, _zeppelin-env.sh_ template, and _zeppelin-site.xml_ template.
2) Also removed values of _env-name_ and _property-name_ (replaced both with `null`) from both _interpreter-setting.json_ files.
3) Renamed the two properties to `zeppelin.spark.sql.interpolation` (for SparkSQL interpreter) and `zeppelin.shell.interpolation` (for Shell interpreter) 
4) Updated documentation files for _configuration_, _spark_, and _shell_.